core/src/main/scala/kafka/utils/ZkUtils.scala

```diff
@@ -240,31 +278,37 @@ object ZkUtils extends Logging {
   /**
    *  make sure a persistent path exists in ZK. Create the path if not exist.
    */
-  def makeSurePersistentPathExists(client: ZkClient, path: String) {
+  def makeSurePersistentPathExists(client: ZkClient, path: String, acls: List[ACL] = DefaultAcls) {
```

Comment:
OK, I looked at this in more detail and it seems like it's the other way around. No-one is passing the `acls` parameter at the moment. So, why is it a parameter? Is it because we think it may be useful in the future? Just trying to understand the reasoning, thanks.